chart/templates/virtualmachine.yaml

```
@@ -0,0 +1,63 @@
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: {{ include "mixmode.fullname" . }}
+  labels:
+    {{- include "mixmode.labels" . | nindent 4 }}
+    kubevirt.io/vm: {{ include "mixmode.fullname" . }}
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: {{ include "mixmode.fullname" . }}
+        {{- include "mixmode.selectorLabels" . | nindent 8 }}
+    spec:
+      domain:
+        devices:
+          disks:
+            - disk:
+                bus: virtio
+              name: {{ include "mixmode.fullname" . }}
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+            - name: default
+              masquerade: {}
+              ports:
+                # Ports not known, including HTTP and SSH for now
+                - port: 22
+                - port: 80
+        resources:
+          requests:
+            memory: 2G # from OVA's OVF XML
+      networks:
+        - name: default
+          pod: {}
+      volumes:
+        - dataVolume:
+            name: {{ include "mixmode.fullname" . }}
+          name: {{ include "mixmode.fullname" . }}
+        - name: cloudinitdisk
+          cloudInitNoCloud:
+            userData: |
+              #cloud-config
+
+              # untested if this works
+  dataVolumeTemplates:
+    - metadata:
+        name: {{ include "mixmode.fullname" . }}
+      spec:
+        pvc:
+          accessModes:
+            - ReadWriteOnce
+          resources:
+            requests:
+              storage: 150Gi  # TODO: add the exact size here
```

```suggestion
              storage: 55Gi  # TODO: add the exact size here
```
The disk is 50 GiB per `qemu-img info`.

At some point we should make sure cloud init grows the disk to fill the volume on first boot. Then slightly overshooting the size is a good thing.